about.html

```diff
@@ -0,0 +1,150 @@
+            >“Our highest deeds come from helping the lowest people.”</span
+          >
+          <h1 class="about-goal">
+            “Overcoming poverty is not a gesture of charity. It is an act of
+            justice. It is the protection of a fundamental human right, the
+            right to dignity and a decent life.”
+          </h1>
+        </div>
+        <div class="about-ideal">
+          The Genuine Charity brings together the community of experts,
+          academics, nad activists who comprise the Creative Commins affiliates
+          network in a different country every two years. This year, we hope to
+          expand our invitation list including organizations and individuals who
+          want to work with us on shared projects taht advance the cause of yhe
+          commons, free culture nd open kwnoledge. So if you're active and
+          engaged in the worlds of open content and kwnoledge - free software
+          advocates. Wikipedias, Open Kwnoledge, galeries, libraries, museums,
+          archives, governments and foundations, kawyers, and activists - we
+          hope you'll consider joing us this year to build a stronger, more
+          vibrant commons together.
+        </div>
+        <div class="contact">
+          Please contact us per Email for any further questions about Genuine
+          Charity and care at
+          <strong><u>christianbiringanine22@gmail.com</u></strong> >
+        </div>
+      </section>
+      <section class="about-logo">
+        <div class="title">
+          <h1>Genuine Charity and Care Logo</h1>
+        </div>
+        <div class="concern">
+          <p>
+            The logo of Guine Charity and care will be decided through the logo
+            competiton form 8 June to 7 July. After online public voting and
+            Selection Committee's judging process. the logo that will win prize.
+            After consultation with Mr. Agrawal will be optimized the
+            prize-winning-logo for the coming years.
+          </p>
+        </div>
+      </section>
+      <section class="past-activities">
+        <h1 class="activ-title">See the past GC campain</h1>
+        <p>
+          Take a look at the last two GC Campain which took place in the last
+          year in Kigali and in Peru
+        </p>
+        <div class="activities-details">
+          <div class="activity1">
+            <div>
+              <span class="date">March</span>
+              <span class="place">GC Campain in Peru</span>
+            </div>
+          </div>
+          <div class="activity2">
+            <div>
+              <span class="date">September</span>
+              <span>GC Campain in Kigali</span>
+            </div>
+          </div>
+        </div>
+      </section>
+      <div class="about-last">
+        <div>
+          <p>&copy; 2022 Genuine Charity. Some Rights Reserved.</p>
+          <p>Created by Christian Biringanine</p>
+        </div>
```

Comment:
- [ ] **Required:** The images loaded into this containers should be `position: center;`

![image](https://user-images.githubusercontent.com/27709832/163601104-03f65ac7-8100-442b-b4ce-fb418faa89e5.png)

Reply:
:100:

Reply:
resolved 👍